client/src/App.js

```diff
@@ -1,13 +1,97 @@
+import {ConfigProvider, Divider, Spin, Typography} from "antd";
+import {useEffect, useState} from "react";
+import "./antd-overrides.css";
+import {myTheme} from "./antd-theme-overrides";
 import "./App.css";
+import AppHeader from "./components/AppHeader";
+import ContentContainer from "./components/ContentContainer";
+import FormAddVideo from "./components/FormAddVideo";
+import SortSelect from "./components/SortSelect";
+import VideoList from "./components/VideoList";
+
+export const API_URL = 'https://video-recomendations-7q29.onrender.com';
+const getAllVideos = (order, setVideos, setLoading) => {
+    setLoading(true);
+    fetch(`${API_URL}?order=${order}`)
+        .then(res => res.json())
+        .then(({videos}) => setVideos(videos))
+        .finally(() => setLoading(false));
+}
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <h1>Video Recommendation</h1>
-      </header>
-    </div>
-  );
+    const [videos, setVideos] = useState([]);
+    const [order, setOrder] = useState('desc');
+    const [loading, setLoading] = useState(false);
+
+    useEffect(() => {
+        getAllVideos(order, setVideos, setLoading);
+    }, [order])
+
+
+    const handleDeleteVideo = (id) => {
+        try {
+            fetch(`${API_URL}/${id}`, {
+                method: "DELETE", // или 'PUT'
+                body: JSON.stringify({id}), // данные могут быть 'строкой' или {объектом}!
```

Comment:
You generally don't need to include the same data in both the path and the body - you can probably just put it in one place or the other, as long as the server knows where it's going to read it from :)